bin/ubtest_common

```diff
@@ -0,0 +1,69 @@
+# Copyright 2020 Google LLC
```

Comment:
This file isn't generic to all possible users. Shouldn't it be in bin/kea/?

Reply:
It is as generic as I can make it, I think. Anyone who wants to use these hooks with servo could use it.

Reply:
Can we name it something that makes it obvious what it's for at least; I guess common-config.cros-servo-dut-control or something like that? At least that way people will know it isn't a truly common file that applies to all users, but rather something only applicable to cases using servo/dutcontrol.

Reply:
OK done